examples/02-HFSS/Probe_Fed_Patch.py

```diff
@@ -0,0 +1,112 @@
+"""
+HFSS: Probe-fed patch antenna
+---------------------------------------------------------
+This example demonstrates the use of the ``Stackup3D`` class
+to create and analyze a patch antenna in HFSS 3D.
+
+Note that the HFSS 3D Layout interface may offer advantages for
+laminate structures such as the patch antenna.
+"""
+
+###########################
+# Perform  imports
+# ~~~~~~~~~~~~~~~~~~
+
+import os
+
+import pyaedt
+import tempfile
+from pyaedt.modeler.advanced_cad.stackup_3d import Stackup3D
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"`` is set to ``False``
+# to create this documentation.
+#
+# You can set ``non_graphical``  to ``True`` in order to view
+# HFSS while the notebook cells are executed.
+#
+# Use the 2023R1 release of HFSS.
```

Comment:
```suggestion
# Use the 2023 R1 release of HFSS.
```
In public-facing doc, the release format should always have a space before the year and the version number.